### ISSUE TITLE
Enhance a11y for `Avatar`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#1259]
path: stories/components/media-and-icons/avatar.stories.tsx

### Description
1 accessibility violation was found in the `Avatar`.

#### [withFallback](https://yamada-ui.github.io/yamada-ui/?path=/story/components-media-and-icons-avatar--with-fallback)

Expected the HTML found at $('.css-lshe7a > .ui-avatar__icon.css-1r6qcyz[viewBox="0 0 128 128"]') to have no violations:
    <svg viewBox="0 0 128 128" class="ui-icon ui-avatar__icon css-1r6qcyz" role="img">

    Received:
    <svg> elements with an img role must have an alternative text (svg-img-alt)

    Fix any of the following:
      Element has no child that is a title
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/svg-img-alt?application=axeAPI

Expected the HTML found at $('.css-w23j27.ui-avatar:nth-child(2) > .ui-avatar__icon.css-1r6qcyz[viewBox="0 0 128 128"]') to have no violations:
    <svg viewBox="0 0 128 128" class="ui-icon ui-avatar__icon css-1r6qcyz" role="img">

    Received:
    <svg> elements with an img role must have an alternative text (svg-img-alt)

    Fix any of the following:
      Element has no child that is a title
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/svg-img-alt?application=axeAPI

### Problem Statement/Justification
A11y is essential to ensure quality when operating a package.

### Additional Information
While it's ideal to cover all jobs, if there are too many jobs to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
